plugins/module_utils/redfish_utils.py

```diff
@@ -3544,3 +3545,112 @@ def delete_volumes(self, storage_subsystem_id, volume_ids):
 
         return {'ret': True, 'changed': True,
                 'msg': "The following volumes were deleted: %s" % str(volume_ids)}
+
+    def create_volume(self, volume_details, storage_subsystem_id):
+        # Fail message if run for iLO5 or lower server
+        vendor = self._get_vendor()['Vendor']
```

Comment:
Why do we need this check up front? Isn't the failure to POST to the VolumeCollection sufficient? Doing version checks for capabilities is not a good practice; the interface itself is self-descriptive

Reply:
I do see your point, but we may benefit from explicitly pointing out that it doesn't work for certain server. Changed the implementation a little, please a have look.